### DISCUSSION TITLE
Cloud Functionality

### DISCUSSION THREAD

**Discussion by master-bob** [charmbracelet/glow#550] [General]
Hello all,

I was curious about the cloud functionality of Glow. From what I can gather glow has a stash feature which stores thing in the cloud. What other features are in the cloud? How does it work in offline mode? And is the cloud feature optional, that is can I use glow without using any cloud functionality?

Thank you for your time and exposition.

**Comment by belibug** [comment]
I am interested to know this too. What exactly does cloud do. How do I manage what is stored on cloud. How to opt out completely.
